TST: import ibis takes more than 2 seconds in Windows

We have a test that checks that `import ibis` doesn't take more than 2 seconds to run. It fails somehow often in Windows.

This will probably be fixed once we start using entrypoints (see #2379), since all backends will be loaded lazily. If it doesn't, we should benchmark what is taking so long to be executed just when importing the module.

```python
______________________________ test_import_time _______________________________

    @pytest.mark.skipif(
        bool(os.environ.get("AZURECI")),
        reason="Testing import time on CI is flaky due to machine variance",
    )
    def test_import_time():
        pb = pytest.importorskip("plumbum")
        lines = ["from timeit import timeit", "print(timeit('import ibis'))"]
        delta = float(pb.cmd.python["-c", "; ".join(lines)]().strip())
>       assert delta < 2.0
E       assert 2.0011241999999996 < 2.0

ibis\tests\test_version.py:18: AssertionError
```

--- a/ibis/tests/test_version.py
+++ b/ibis/tests/test_version.py
@@ -8,7 +8,7 @@ import ibis
 
 
 @pytest.mark.skipif(
-    bool(os.environ.get("AZURECI")),
+    bool(os.environ.get("CI")),
     reason="Testing import time on CI is flaky due to machine variance",
 )
 def test_import_time():

